man/nextcloudcmd.1.rst

```diff
@@ -32,46 +32,56 @@ OPTIONS
 ``--path``
        Overrides default remote root folder to a specific subfolder on the server(e.g.: /Documents would sync the Documents subfolder on the server)
 
-``—user``, ``-u`` ``[user]``
+``--user``, ``-u`` ``[user]``
```

Comment:
This makes me wonder, does `-user` still work? If not it might be a deal breaker for the change in nextcloudcmd since the exposed CLI can't change there, people are likely using this in scripts and so it'd break them.

Reply:
we already had people having to adjust for even more minor workflow changes of `nextcloudcmd`
it would be best to still offer compatibility maybe with a deprecation warning (and I know that if run from a script nobody will read it)

Reply:
the current code is already using `--user` and not `-user` so that should be fine to do